src/solvers/cma_es.jl

```diff
@@ -23,12 +23,12 @@ State of covariance matrix adaptation evolution strategy.
 * `population`                  population of the current generation
 * `ys_c`                        coordinates of random vectors for the current generation
 * `covariance_matrix`           coordinates of the covariance matrix
-* `covariance_matrix_eigen`     eigendecomposition of `covariance_matrix`
-* `covariance_matrix_cond`      condition number of `covariance_matrix`, updated after eigendecomposition
+* `covariance_matrix_eigen`     eigen decomposition of `covariance_matrix`
```

Comment:
What's wrong with `eigendecomposition` written as one word? Wikipedia doesn't split it into two words.

Reply:
In German I would definetly put that together, in English I am not sure, and I probably changed it because vale complained, we can surely change that back and add that word to the `accepted.txt` lis in the docs

Reply:
(or maybe I was also a bit tired after 100-or-so fixes from vale and note careful enough, _but_ vale even already passes on this PR and will be introduced on CI somewhen soon)

Reply:
I've added eigendecomposition to accepted.txt list.